buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy

```diff
@@ -452,6 +454,41 @@ class VagrantTestPlugin implements Plugin<Project> {
                 command BATS_TEST_COMMAND
             }
             packagingTest.dependsOn(packaging)
+
+            String task = "vagrant${boxTask}#repositoryTest"
+
+            String expectedRepoVersion = project.getProperties().get('repo.version')
+            String url = project.getProperties().get('repo.url')
+
+            Task createUrlFileTask = project.tasks.create("vagrant${boxTask}#createUrlFileTask", FileContentsTask) {
+                file "${batsDir}/archives/repo_url"
```

Comment:
Probably.... But if we are doing it twice already I'm fine with doing it one more time. We can clean them all up at some point. Or I'm forgetting something and there is a really good reason to do it like we do it. Either way, I withdraw this objection.